docs/access/jupyterlab.md

```diff
@@ -0,0 +1,238 @@
+    As a result, JupyterLab may take slightly longer than usual to start.
+
+!!! warning "Ending your interactive session and logging out"
+    The Jupyter servers can be shut down through the Hub. To end a JupyterLab session, please select `Hub Control Panel` under the `File` menu and then `Stop My Server`. By contrast, clicking `Logout` will log you out of the server, but the server will continue to run until the Slurm job reaches its maximum wall time.
+
+If the default base images do not meet your requirements, you can specify a custom environment instead. For this purpose, you supply either a custom uenv image/view or CE TOML file under the section `Advanced options` before launching the session. The supported uenvs are compatible with the Jupyter service out of the box, whereas container images typically require the installation of some additional packages. 
+
+??? "Example of a custom Pytorch container"
+    A container image based on recent a NGC Pytorch release requires the installation of the following additional packages to be compatible with the Jupyter service:
+
+    ```Dockerfile
+    FROM nvcr.io/nvidia/pytorch:25.05-py3
+
+    RUN pip install --no-cache \
+        jupyterlab \
+        jupyterhub==4.1.6 \
+        pyfirecrest==1.2.0 \
+        SQLAlchemy==1.4.52 \
+        oauthenticator==16.3.1 \
+        notebook==7.3.3 \
+        jupyterlab_nvdashboard==0.13.0 \
+        git+https://github.com/eth-cscs/firecrestspawner.git
+    ```
+
+    The package [nvdashboard](https://github.com/rapidsai/jupyterlab-nvdashboard) is also installed here, which allows to monitor system metrics at runtime.
+    
+    A corresponding TOML file can look like
+
+    ```toml
+    image = "/capstor/scratch/cscs/${USER}/ce-images/ngc-pytorch+25.05.sqsh"
+
+    mounts = [
+        "/capstor", 
+        "/iopsstor",
+        "/users/${USER}/.local/share/jupyter", # (1)!
+        "/etc/slurm", # (2)!
+        "/usr/lib64/libslurm-uenv-mount.so",
+        "/etc/container_engine_pyxis.conf" # (3)!
+    ]
+
+    workdir = "/capstor/scratch/cscs/${USER}" # (4)!
+
+    writable = true
+
+    [annotations]
+    com.hooks.aws_ofi_nccl.enabled = "true" # (5)!
+    com.hooks.aws_ofi_nccl.variant = "cuda12"
+
+    [env]
+    CUDA_CACHE_DISABLE = "1" # (6)!
+    TORCH_NCCL_ASYNC_ERROR_HANDLING = "1" # (7)!
+    MPICH_GPU_SUPPORT_ENABLED = "0" # (8)!
+    ```
+    
+    1. avoid mounting all of `$HOME` to avoid subtle issues with cached files, but mount Jupyter kernels
+    2. enable SLURM commands (together with two subsequent mounts)
+    3. currently only required on Daint and Santis, not on Clariden
+    4. set working directory of Jupyter session (file browser root directory)
+    5. use environment settings for optimized communication 
+    6. disable CUDA JIT cache
+    7. async error handling when an exception is observed in NCCL watchdog: aborting NCCL communicator and tearing down process upon error
```

Comment:
```suggestion
    1. Avoid mounting all of `$HOME` to avoid subtle issues with cached files, but mount Jupyter kernels
    2. Enable SLURM commands (together with two subsequent mounts)
    3. Currently only required on Daint and Santis, not on Clariden
    4. Set working directory of Jupyter session (file browser root directory)
    5. Use environment settings for optimized communication 
    6. Disable CUDA JIT cache
    7. Async error handling when an exception is observed in NCCL watchdog: aborting NCCL communicator and tearing down process upon error
```